lib/utils/setPageCookies.js

```diff
@@ -0,0 +1,23 @@
+const debug = require('./debug');
+
+/**
+ * @param {!RemoteBrowser} context
+ * @param {string} url
+ */
+
+const setPageCookies = async function setPageCookies(context, url) {
+  try {
+    const { hostname } = new URL(url);
+
+    if (context.cookiesQueue.length) {
+      await context.page.setCookie(...context.cookiesQueue.map(cookie => ({
+        ...cookie,
+        domain: hostname,
```

Comment:
We probably want to change the order here to:

```js
{
  domain: hostname,
  ...cookie,
}
```

This way we will be able to redefine the domain in the passed cookie object. E.g. to set the cookie for `*.domain.com` instead of `sub.domain.com`, etc.

Not sure that this is important, btw.

Reply:
The cookie object does not use the `domain` field, but if it is added in the future, no fix will need to be made here.